### PR TITLE
Lola's Cupcakes

### PR DESCRIPTION
**Pull request by codeinabox**

### CODE REVIEW

**Comment by codeinabox** [comment]
@peternewman keen to hear your feedback on this as I noticed you've tagged a few of their stores

**Review by LaoshuBaby — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks @codeinabox 👍

**Comment by peternewman** [comment]
> @peternewman keen to hear your feedback on this as I noticed you've tagged a few of their stores

I didn't realise I'd done that many! :laughing: Haven't eaten from them much sadly!

File: data/brands/shop/pastry.json [hunk 1]
```diff
@@ -122,6 +122,18 @@
         "shop": "pastry"
       }
     },
+    {
+      "displayName": "Lola's Cupcakes",
+      "locationSet": {
+        "include": ["gb-lon.geojson"]
```
Comment (peternewman) [inline data/brands/shop/pastry.json:128]: Despite the website saying London a lot, they definitely have stores listed in Milton Keynes, Cambridge and Birmingham, so I think this should be gb-eng at the tightest:
https://www.lolascupcakes.co.uk/cupcakes/stores

There are also two international franchises listed in Japan:
https://www.lolascupcakes.co.uk/InternationalStores.aspx

**Comment by peternewman** [comment]
They also have lockers now that we should probably add:
https://www.lolascupcakes.co.uk/cupcakes/lolaslockers

**Pull request merged**